packages/server/src/routes/members/index.ts

```diff
@@ -540,64 +542,154 @@ export const give = (
 
 export const createMember = (
```

Comment:
TODO, leave a comment explaining this; talked offline, it's a somewhat complex thing so we should clarify